module/userProjects/helper.js

```diff
@@ -555,6 +555,108 @@ module.exports = class UserProjectsHelper {
 		})
 	}
 
+	/**
+	 * Program and solution information
+	 * @method
+	 * @name createProgramAndMultipleSolutions
+	 * @param {Object} programData - program data
+	 * @param {Array} solutionData - solutions data
+	 * @param {String} userId - user id
+	 * @param {Boolean} isAPrivateProgram - isAPrivateProgram
+	 * @returns {Object} Created program and solution data.
+	 */
+
+	static createProgramAndMultipleSolutions(programData, solutionData, userId, isAPrivateProgram = false) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				const dateFormat = UTILS.epochTime()
+				let startDate = new Date()
+				let endDate = new Date()
+				endDate.setFullYear(endDate.getFullYear() + 1)
+				let userPrivateProgram
+
+				// Fetch programs data
+				if (programData._id && programData._id != '') {
+					userPrivateProgram = await programQueries.programsDocument(
+						{
+							_id: programData._id,
+						},
+						'all'
+					)
+					userPrivateProgram = userPrivateProgram[0]
+				} else {
+					// Create a program if program id is not passed
+					let program = await solutionsHelper._createProgramData(
+						programData.name,
+						programData.externalId ? programData.externalId : programData.name + '-' + dateFormat,
+						isAPrivateProgram,
+						CONSTANTS.common.ACTIVE_STATUS,
+						programData.description ? programData.description : programData.name,
+						userId,
+						programData.startDate ? programData.startDate : startDate,
+						programData.endDate ? programData.endDate : endDate,
+						userId,
+						programData.language ? programData.language : [],
+						programData.conversation ? programData.conversation : [],
+						programData.source ? programData.source : {}
+					)
+					userPrivateProgram = await programQueries.createProgram(program)
+				}
+
+				let solutionsCreated = []
+				for (const item of solutionData) {
+					// Create a solution if solutions
+					let solution = await solutionsHelper._createSolutionData(
+						item.name ? item.name : programData.name,
+						item.externalId ? item.externalId : programData.name + '-' + dateFormat,
+						isAPrivateProgram,
+						CONSTANTS.common.ACTIVE_STATUS,
+						item.description != '' ? item.description : programData.name,
+						userId,
+						false,
+						'',
+						'improvementProject',
```

Comment:
@Prajwal17Tunerlabs We will have constant for this value please check

Reply:
Replaced with constant.